nemo_curator/package_info.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+MAJOR = 0
+MINOR = 6
+PATCH = 0
+PRE_RELEASE = "rc0"
```

Comment:
Again I'm not sure what the convention is, but I prefer leaving this empty `""` and just having `DEV` until code freeze or QA.

Reply:
No, both segments (prerelease and dev) are filled. For the actual prerelease workflow that tags a commit, we‘ll set dev to "". And for the actual release workflow that tags a commit we‘ll set both dev and prelease to ""